api/envoy/extensions/filters/http/oauth2/v3/oauth.proto

```diff
@@ -149,6 +149,10 @@ message OAuth2Config {
   // in a week.
   // This setting is only considered if ``use_refresh_token`` is set to true, otherwise the authorization server expiration or ``defaul_expires_in`` is used.
   google.protobuf.Duration default_refresh_token_expires_in = 15;
+
+  // If set to true, preserve the existing authorization header when forward_bearer_token is false.
+  // Default value is false.
+  google.protobuf.BoolValue preserve_authorization_header = 16;
```

Comment:
simple `bool` should be enough. And please move this field to below of `forward_bearer_token` because they are related to each others.